From 4b79840af6b4fddaef412826087fc6ea354d0a33 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Tue, 2 May 2017 09:48:52 -0700
Subject: [PATCH] MANTA-3239 marlin agent crash after invalid cueball state
 transition Reviewed by: Kody A Kantor <kody.kantor@gmail.com> Approved by:
 Robert Mustacchi <rm@joyent.com>

---
 dev/package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dev/package.json b/dev/package.json
index 46d11e4..ff2b4cb 100644
--- a/dev/package.json
+++ b/dev/package.json
@@ -16,7 +16,7 @@
 		"assert-plus": "^1.0.0",
 		"bunyan": "0.20.0",
 		"catstreams": "0.5.0",
-		"cueball": "2.2.4",
+		"cueball": "2.2.6",
 		"dtrace-provider": "0.6.0",
 		"extsprintf": "^1.3.0",
 		"fuzzstream": "0.0.1",
-- 
2.21.0

